modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js

```diff
@@ -183,10 +184,10 @@ const FieldCategory = ({categoryName}) => (
 	</div>
 );
 
-export default ({keywords}) => {
+const CustomObjectFieldsList = ({keywords}) => {
```

Comment:
@interaminense @kevenleone, We're avoiding export as default directly anonymous functions/classes because We can't track these components properly on React DevTools. See:

Before:

![image](https://user-images.githubusercontent.com/7663513/107668870-e7bc9e80-6c6f-11eb-9a07-2eda4b86d629.png)


Would be a good idea to consider this guideline on your components(DE/App Builder)for avoiding these `_default` components

Reply:
Totally agreed with you, @diegonvs, thanks in advance

I'll also, share with the rest of App Builder frontends

Reply:
@diegonvs, thanks for your feedback!